### ISSUE TITLE
Update display of BCID links

### ISSUE THREAD

**Issue by jdeck88** [biocodellc/geome-ui#447]
On specimen, event, tissue, FASTQ, diagnostic (all entities!) pages we display BCIDs but the show up in the form like:

```
ark:/21547/CgZ2PEER_1878
```
but they should be:
```
https://n2t.net/ark:/21547/CgZ2PEER_1878
```
See example image below:
![image](https://user-images.githubusercontent.com/1154501/80036946-91475f00-84a7-11ea-9d37-18f6f64adb0f.png)

**Comment by jdeck88** [comment]
Also, the tissueCatalogNumber in the image should be a hyperlink....

**Comment by albarqmaria** [comment]
also, project link returns 404

**Comment by albarqmaria** [comment]
Fixed in PR #460
